docs/Plugins/refdiff.md

```diff
@@ -89,6 +89,28 @@ curl -v -XPOST http://localhost:8080/pipelines --data @- <<'JSON'
 JSON
 ```
 
+4. Refdiff has a new subtask `calculateProjectDeploymentCommitsDiff`, which can find related deployments according to the project, and then calculate the commits diff between deployments. Such a function can be used to support the calculation of dora indicators.
```

Comment:
Text change:
"Such a function can be used to support the calculation of dora indicators." to 
"This function can be used to support the calculation of DORA metrics."